framework/interface/api_handlers.py

```diff
@@ -52,38 +52,20 @@ def get(self, plugin_group=None, plugin_type=None, plugin_code=None):
 class PluginNameOutput(custom_handlers.UIRequestHandler):
     SUPPORTED_METHODS = ['GET']
 
-    def get(self, target_id=None, plugin_group=None, plugin_type=None, plugin_code=None):
+    def get(self, target_id=None):
```

Comment:
@saganshul Thanks for taking the time to add documentation. However I feel that the documentation you added for this API is incorrect in several ways that I would like to present:

+ The rest API path should not be listed in the docstring since both are independent. We could imagine moving API around, which would make this doc outdated
+ How the returned data is being used should not be listed either since it is also independent from the function itself

Instead, I feel that a docstring explaining what information is generated and returned would be better. You started explaining that on the second line with `This API returns information about a target scan present in OWTF except `output` section.`. It would be great if you detail that a bit further.

It would be better to have the docstring explains what is returned and how the data is being processed (if there is any processing).

Could you attempt to use Sphinx's syntax for the docstring? Something similar to https://github.com/owtf/ptp/blob/develop/ptp/libptp/parser.py#L57 for instance.